WordPress/Classes/ViewRelated/Me/App Settings/DebugMenuViewController.swift

```diff
@@ -39,41 +37,25 @@ class DebugMenuViewController: UITableViewController {
     }
 
     private func reloadViewModel() {
-        let remoteFeatureFlags = RemoteFeatureFlag.allCases.filter({ $0.canOverride })
-        let remoteFeatureFlagsRows: [ImmuTableRow] = remoteFeatureFlags.map({ makeRemoteFeatureFlagsRows(for: $0) })
-
-        let localFeatureFlags = FeatureFlag.allCases.filter({ $0.canOverride })
-        let localFeatureFlagsRows: [ImmuTableRow] = localFeatureFlags.map({ makeLocalFeatureFlagsRow(for: $0) })
-
         handler.viewModel = ImmuTable(sections: [
-            ImmuTableSection(headerText: Strings.remoteFeatureFlags, rows: remoteFeatureFlagsRows),
-            ImmuTableSection(headerText: Strings.localFeatureFlags, rows: localFeatureFlagsRows),
+            ImmuTableSection(headerText: "General", rows: generalRows),
```

Comment:
Have considered making these new strings localized?

One could argue that since this is an internal menu and does not appear in public builds, it's okay to have it in English only. 

At the same time, adding localizations now is cheaper and safer than adding them in the future. They're a handful of strings that the community translators should have no problem handling.

Reply:
Yes, I removed the localization so that it doesn't confuse the translators. But it's not ideal that most of the screen is localized, and this new section isn't, so I added localization here as well. I would preferably remove localization in the entire debug menu.

Reply:
@kean can you elaborate on what you mean with "confusing the translators"?

I think you mentioned it in another PR and I want to be sure I understand it.

In my experience so far, the open source community that helps localize the apps is seldom confused. I only seen them confused once when we put some made up words that made sense only in English to add a bit of humor in the copy.

By the way, all the work happens in the open, of course, and you can find them in [the `#polyglots` channel in the Making WordPress Slack](https://wordpress.slack.com/archives/C02RP50LK).

Reply:
> can you elaborate on what you mean with "confusing the translators"?

It's likely that the debug screens would have some obscure technical terms or phrases that are hard to interpret without knowing the behind-the-scenes of the app. Unless there is a need to localize them, I would rather conserve the resources of both the translators and the developers: adding localization slightly increases the dev effort.

Reply:
> It's likely that the debug screens would have some obscure technical terms or phrases that are hard to interpret without knowing the behind-the-scenes of the app.

🤔 You might be underestimating the tech savviness of the community working on this. 

> Unless there is a need to localize them, I would rather conserve the resources of both the translators and the developers: adding localization slightly increases the dev effort.

We can happily live with an English only debug screen, no dramas. 

But I think it's be best to assume _everything_ should be localized, as a rule of thumb.  WordPress's ethos is to democratize publishing. Localization is part of that. 

I see the overhead of developers wrapping raw strings into their localization harness and of translators translating them negligible when compared to the end product's reach in our user base.